index.js

```diff
@@ -350,3 +344,33 @@ function sqSegSegDist(x0, y0, x1, y1, x2, y2, x3, y3) {
 
     return dx * dx + dy * dy;
 }
+
+function cross(p1, p2, p3) {
+    return orient(p1[0], p1[1], p2[0], p2[1], p3[0], p3[1]);
+}
+
+function convexHull(points) {
+    points.sort(function (a, b) {
+        return a[0] === b[0] ? a[1] - b[1] : a[0] - b[0];
+    });
```

Comment:
Nit: I prefer to not inline anonymous functions wherever possible (this one could be extracted as `compareByX` to the top level).